src/seedu/addressbook/AddressBook.java

```diff
@@ -124,6 +124,14 @@
     private static final String COMMAND_HELP_DESC = "Shows program usage instructions.";
     private static final String COMMAND_HELP_EXAMPLE = COMMAND_HELP_WORD;
 
+	private static final String COMMAND_SORT_WORD = "sort";
```

Comment:
Indentation with tabs instead of spaces here (and for the rest of this PR). You can configure eclipse to use spaces for indentation instead of tabs: http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs